charts/akeyless-secure-remote-access/templates/deployment.yaml

```diff
@@ -69,6 +69,14 @@ spec:
             - name: USERNAME_SUB_CLAIM
               value: {{ .Values.usernameSubClaim }}
 {{- end }}
+{{- if .Values.ztbConfig.usernameSubClaim }}
+            - name: RDP_USERNAME_SUB_CLAIM
```

Comment:
does the RDP_USERNAME_SUB_CLAIM and SSH_USERNAME_SUB_CLAIM are the same value? or they can be different for each service (ssh and ztb)?

Reply:
In current version it is the same valuein USERNAME_SUB_CLAIM, and once it is set, it is set for both.
In new version it can be different or can be set only for one of them and the other won't get an unwanted value.
USERNAME_SUB_CLAIM is kept for BC.